Apps/W1/Shopify/app/src/Base/Pages/ShpfyShopCard.Page.al

```diff
@@ -974,6 +997,25 @@ page 30101 "Shpfy Shop Card"
                         Report.Run(Report::"Shpfy Sync Shipm. to Shopify");
                     end;
                 }
+                action(SyncPostedSalesInvoices)
```

Comment:
I think this action should include the propery `Ellipsis=true`, since we're opening a report request page.